tests/models.py

```diff
@@ -269,7 +269,7 @@ class Meta:
     published_by = factory.Maybe("published", yes_declaration=factory.SelfAttribute("_publisher.email"))
     unpublished_at = factory.Maybe("_unpublished", yes_declaration=Faker().past_date(start_date="-7d"))
     unpublished_by = factory.Maybe("_unpublished", yes_declaration=factory.SelfAttribute("_unpublisher.email"))
-    db_version_id = factory.Sequence(lambda x: x)
+    # db_version_id = factory.Sequence(lambda x: x)
```

Comment:
We probably do want this set in fixtures generally? And if not then we should just delete it rather than comment it out... maybe a comment in the code to explain why we don't have it in the factory?

Reply:
Hmm, guess my commit message is not clear enough.

This field is slightly deceptive to have in the factory. When you create the instance and flush it to the database, regardless of what you've set the value of `db_version_id` to initially, it always gets reset by SQLAlchemy to 1. It's not something we really need to fake, and it's also not something we probably *want* to fake. Faking it will give us less confidence that it's doing what we expect it to do.

Reply:
I reckon if we're leaving this commented out in the code we should have a short note by it to explain why it's not a good idea to fake it.

Reply:
👍